evaluate.py

```diff
@@ -55,46 +56,51 @@ def generate(args, g_ema, device, mean_joints, std_joints, entity):
         index = range(len(generated_motion))
 
     generated_motion_np, _ = get_gen_mot_np(args, generated_motion, mean_joints, std_joints)
-    generated_motions = np.concatenate(generated_motion_np, axis=0)
 
-    if entity.str() == 'Joint':
```

Comment:
what happens if entity is Joint? like in the model trained on Humanact12

Reply:
Natan, following Inbal's comment, you can apply this evaluation script on the humanact12 trained model (see one of the latest issues to find it). Also, apply on a MoDi trained model. For both cases, make sure the paper's metrics are reconstructed.

Reply:
I fixed it back.